packages/web-components/src/horizontalbarchart/horizontalbarchart.stories.ts

```diff
@@ -13,6 +13,12 @@ export default {
 
 export const Basic: Story<FluentHorizontalBarChart> = renderComponent(storyTemplate).bind({});
 
+export const NMVariant: Story<FluentHorizontalBarChart> = renderComponent(html<StoryArgs<FluentHorizontalBarChart>>`
+  <div>
+    <fluent-horizontalbarchart variant="part-to-whole"> </fluent-horizontalbarchart>
```

Comment:
use this as an enum similar to absolute variant

Reply:
Enum is already defined. This is passed as an attribute similar to prop that we used in React.